data/json/recipes/food/dry.json

```diff
@@ -331,7 +331,7 @@
     "autolearn": true,
     "batch_time_factors": [ 67, 5 ],
     "tools": [ [ [ "dehydrator", 25 ], [ "char_smoker", 25 ] ], [ [ "surface_heat", 5, "LIST" ] ] ],
-    "components": [ [ [ "eggs_bird", 12, "LIST" ], [ "egg_reptile", 12 ], [ "spider_egg", 4 ], [ "ant_egg", 1 ], [ "egg_roach", 4 ] ] ]
+    "components": [ [ [ "eggs_bird", 12, "LIST" ], [ "egg_reptile", 12 ], [ "eggs_insect", 1, "LIST" ] ] ]
```

Comment:
Currently this would produce the equivalent of 12 reptile or bird eggs in powdered eggs from one ant egg or four of most of the rest.  Are the giant insect eggs the size of 3 bird eggs each? If true it will likely through off recipes all over the place.

Reply:
i am not sure how this will affect the balance of the game, i have only encountered centipedes so far.
the egg description of most insects describe them as "fist-sized" which is definately larger than an egg or two.
i only wanted to put them in a recipe because they have literally no use anywhere else but a danger to spawn hostile hatchlings.
the description of the abstract insect egg in json/items/comestibles/egg.json describes them as "5x the weight and 3x the nutrition of real chicken eggs, toxins equal 296g mutant meat"
what do you propose? perhaps add more toxins to the egg?